VMEncryption/test/test_crypt_mount_config_util.py

```diff
@@ -0,0 +1,200 @@
+import unittest
+import mock
+
+from main.Common import CommonVariables, CryptItem
+from main.EncryptionEnvironment import EncryptionEnvironment
+from main.CryptMountConfigUtil import CryptMountConfigUtil
+from console_logger import ConsoleLogger
+from StringIO import StringIO
+
+
+class Test_crypt_mount_config_util(unittest.TestCase):
```

Comment:
is there a way to figure out how much code coverage the tests are giving? Might be a good indicator/tool to run.

Reply:
I don't yet have a tool to do that..
Will probably add that to our toolset later